tutorials/space_shooter/README.md

```diff
@@ -1,3 +1,55 @@
-# Flame tutorial
+# Flame Tutorial
+ABOUT:
+In this Flame tutorial we will provide you with the foundational steps needed to build several parts of the engine of the space shooter game.
+
+This README contains useful videos and articles from the community for this project. The links for these can be found under "Highlighted Community Tutorials."
+
+*Installation information*
+
+<ul>
+    <li>Need Flutter SDK installed for Android platform</li>
+    <li>Feel Free to use an IDE of your choice.</li>
+</ul>
+
+# Part 1
+
+We will teach you how to create a game class, insert it into the Flutter widget tree, and render a simple component.
+
+A simple tutorial to get started can be found on
+[tutorials.flame-engine.org](https://tutorials.flame-engine.org) and examples of most features in
+Flame can be found on [examples.flame-engine.org](https://examples.flame-engine.org). To access the
+code for each example, press the `< >` button in the top right corner.
+
+ 
+## Highlighted community tutorials
+
+- @Devowl's Flutter & Flame series:
+  - [Step 1: Create your game](https://medium.com/flutter-community/flutter-flame-step-1-create-your-game-b3b6ee387d77)
+  - [Step 2: Game basics](https://medium.com/flutter-community/flutter-flame-step-2-game-basics-48b4493424f3)
+  - [Step 3: Sprites and inputs](https://blog.devowl.de/flutter-flame-step-3-sprites-and-inputs-7ca9cc7c8b91)
+  - [Step 4: Collisions & Viewport](https://blog.devowl.de/flutter-flame-step-4-collisions-viewport-ff2da048e3a6)
+  - [Step 5: Level generation & camera](https://blog.devowl.de/flutter-flame-step-5-level-generation-camera-62a060a286e3 )
+
+- Other tutorials:
+  - @Vguzzi's article [Building Games in Flutter with Flame](https://www.raywenderlich.com/27407121-building-games-in-flutter-with-flame-getting-started)
+  - @DevKage's YouTube series with the [Dino run tutorial](https://www.youtube.com/playlist?list=PLiZZKL9HLmWOmQgYxWHuOHOWsUUlhCCOY)
+  - @RutviShukla's Youtube tutorial on [SpaceShooter Pt.1](https://youtu.be/VEfZgTGolPY)
```

Comment:
I appreciate that more people are creating content around Flame, but this video feels more suitable for [awesome-flame](https://github.com/flame-engine/awesome-flame) rather than a highlight tutorials

Reply:
Maybe all these should be moved to awesome flame actually, and then we can just link to awesome flame from here?

Reply:
Yeah, that would be good. This readme shouldn't point to anything outdated/unofficial (mine is quite outdated 😅 anyways)